src/hotspot/share/opto/library_call.cpp

```diff
@@ -8234,6 +8238,27 @@ bool LibraryCallKit::inline_fp_min_max(vmIntrinsics::ID id) {
   return true;
 }
 
+//------------------------------inline_long_min_max------------------------------
+bool LibraryCallKit::inline_long_min_max(vmIntrinsics::ID id) {
+  assert(callee()->signature()->size() == 4, "minL/maxL has 2 parameters of size 2 each.");
+  Node *a = argument(0);
```

Comment:
```suggestion
  Node* a = argument(0);
```
And the same for `b` and `n` as well.